getting-started/deployment_guide.md

```diff
@@ -94,6 +96,69 @@ karmor log
 ## Prerequisites
 1. [K3s](../deployments/k3s)
 2. [MicroK8s](../contribution/microk8s)
-3. [Minikube](../contribution/minikube#minikube-installation)
+  <details>
+3. <summary>Minikube</summary>
+    If virtualbox and vagrant are not installed on your machine, please the following commands in advance.
+
+    ```
+    $ cd KubeArmor/contribution/minikube
+    ~/KubeArmor/contribution/minikube$ ./install_virtualbox.sh
+    ~/KubeArmor/contribution/minikube$ sudo reboot
+    ```
+
+    After rebooting the machine, please keep running the following commands.
+
+    ```
+    $ cd KubeArmor/contribution/minikube
+    ~/KubeArmor/contribution/minikube$ ./install_minikube.sh
+    ```
+
+    Ensure to use virtualbox driver when running minikube. This step is necessary in order to mount roofs as read/write.
+
+    ```
+    $ minikube config set driver virtualbox
+    ```
+
+    In order to use KubeArmor, Minikube needs to support eBPF capabilities. Unfortunately, Minikube doesn't suport them by default. We have compiled Minikube's Kernel with eBPF capablities and AppArmor which is required to enforce security policies. Thus, please run the following command rather than simply running "minikube start".
+
+    ```
+    ~/KubeArmor/contribution/minikube$ ./start_minikube.sh
+    ```
+
+    It will use the minikube image with Linux kernel 5.4.40 with AppArmor service enabled by default.
+
+    If you see no error, you're ready to test KubeArmor.
+  </details>
 4. [Self-managed K8s](../contribution/self-managed-k8s)
 5. [Amazon Elastic Kubernetes Service (EKS)](../deployments/EKS#prerequisite-for-the-deployment)
+6. Setting Environment for KubeArmor
+
+<details>
+  <summary>To set up KubeArmor using MiniKube cluster</summary>
+  
+  If virtualbox and vagrant are not installed on your machine, please run the following commands in advance.
+  ```
+  $ cd KubeArmor/contribution/minikube
+  ```
+  ~/KubeArmor/contribution/minikube$ ./install_virtualbox.sh
+  ~/KubeArmor/contribution/minikube$ sudo reboot
+  After rebooting the machine, please keep running the following commands.
+
+  ```
```

Comment:
This is duplicate of the above right?

Reply:
Let's just get this removed then we should be good to go!

Reply:
Which? I can't able to get you

Reply:
The 6th bullet point, It seems to be same as the 3rd one.

Reply:
Yes, it is. What changes should I make now

Reply:
Just remove it i guess, Any particular reason you added the 6th point?

Reply:
Sorry, At first I can't able to figure out where to add that point so, I added it at last